src/llv8.cc

```diff
@@ -385,8 +392,19 @@ std::string JSFunction::Inspect(InspectOptions* options, Error& err) {
 
     Context context(context_obj);
 
+    ss << rang::fg::magenta << res << rang::fg::reset;
+    res = ss.str();
```

Comment:
I would rather have scoped stringstreams instead of clearing it every time (there are other places this pattern is used if you decide to change it):

```
  {
    std::stringstream ss;
    ss << rang::fg::magenta << res << rang::fg::reset;
    res = ss.str();
  }
```